README.md

```diff
@@ -6,3 +6,29 @@
 [![CircleCI status](https://circleci.com/gh/lacework/terraform-gcp-service-account.svg?style=shield)](https://circleci.com/gh/lacework/terraform-gcp-service-account)
 
 Terraform module that creates a service account to provide Lacework read-only access to Google Cloud Platform Organizations and Projects.
+
+## Simple Usage
+
+### For Compliance Integration
```

Comment:
Is this only for compliance assessments, or also for audit trail? 🤔

Reply:
Only compliance. The roles for Audit Trail exist inside the module https://github.com/lacework/terraform-gcp-audit-log/blob/main/main.tf#L23-L31